### ISSUE TITLE
OctoWS2811 adaptor board?

### ISSUE THREAD

**Issue by Britman72** [DirectOutput/TeensyStripController#3]
Do you absolutely need the OctoWS2811 adaptor board for this to work? Specifically, I want to connect a strip of addressable APA102s direct to the Teensy and control them. Teensy does work with them, but does TeensyStripController work without the Octo board?

**Comment by MikePinball** [comment]
Yes and no. You need the circuitry that is on the OctoWS2811 board to interface between the 3.3V Teensy and the 5V for the LEDs. You can see this by examining the [schematic](https://www.pjrc.com/store/schematic_octo28.gif).

I have built my own version of the Octo board which I call the [Pinball Addressable LED (PAL)](https://www.vpforums.org/index.php?showtopic=43482) board. It is described and sold on VPForums.org and has a [user guide](https://drive.google.com/open?id=1Zk_5RxsWX4VIPhT4XtGlj1rNlBTug39a). My PAL board has some advantages over the Octo board including a built-in test function and that it is programmed ready to go.

![pal_board](https://user-images.githubusercontent.com/41599843/74076224-46579880-49dc-11ea-9bd3-d1803e37dc71.jpg)

**Comment by Rappelbox** [comment]
> 
> 
> Do you absolutely need the OctoWS2811 adaptor board for this to work? Specifically, I want to connect a strip of addressable APA102s direct to the Teensy and control them. Teensy does work with them, but does TeensyStripController work without the Octo board?

A simple to use solution with a casing can also be found here:
[https://germangamingsupplies.com/Teensy-Control-Box](url)

It has 4 prewired outputs, comes with instructions manual, config files and has a nice case to protect the circuit against shorts.
![grafik](https://user-images.githubusercontent.com/15606530/79249701-f2ef4400-7e7d-11ea-9663-90abc3591ff8.png)
